op-guide/docker-across-multiple-hosts.md

```diff
@@ -0,0 +1,166 @@
+# Run TiDB in Docker across Multiple Hosts
+
+This page shows you a example of manually deploying a TiDB cluster using docker on multiple machines.
+
+
+## Preparation
+Assume we have 3 machines with the following details:
+
+|Name|Host IP|
+|----|-------|
+|**host1**|192.168.1.100|
+|**host2**|192.168.1.101|
+|**host3**|192.168.1.102|
+
+Every host has already installed newest version docker, and pulled the latest docker images of TiDB/TiKV/PD.
```

Comment:
+On each host, we have installed Docker with the latest version and pulled the latest Docker images of TiDB,TiKV and PD. (Please include the docker hub address as Shen Li suggested.)